src/Plugin.cc

```diff
@@ -290,24 +309,35 @@ std::string Plugin::ConfigStr()
   if (!pluginElem)
   {
     gzerr << "Missing <plugin> element, not updating config string."
-           << std::endl;
+          << std::endl;
     return this->configStr;
   }
 
-  // <ignition-gui>
-  auto ignGuiElem = pluginElem->FirstChildElement("ignition-gui");
-  if (!ignGuiElem)
+  // <gz-gui>
+  auto guiElem = pluginElem->FirstChildElement("gz-gui");
+  if (!guiElem)
   {
-    ignGuiElem = doc.NewElement("ignition-gui");
-    pluginElem->InsertEndChild(ignGuiElem);
+    // TODO(CH3): Deprecated. Remove on tock.
```

Comment:
This function is used for saving the config, so I think just changing `ignition` to `gz` is enough. No need to tick-tock. All new saved configs will have `gz`, which is ok. The tick-tock you did for loading old configs that contain `ignition` is enough to tick-tock this feature.

Reply:
https://github.com/gazebosim/gz-gui/pull/413/commits/fe598592f38834a889c2d6bde41bcaee30b3c1d5 un-tick-tocked!